pages/services/edge-lb/1.4/cloud-connector/index.md

```diff
@@ -0,0 +1,652 @@
+
+- `DC/OS:EdgeLB:LoadBalancerName` that specifies the original load balancer name. For example: echo.
+
+Target groups have two additional tags:
+
+- `DC/OS:EdgeLB:FrontendName` that specifies the name of a corresponding frontend. For example: echo.
+
+- `DC/OS:EdgeLB:ListenerPort` that specifies a port number. For example: 80.
+
+## Required permissions
+Either the instance on which Edge-LB API server is running on, or the IAM user that is specified using an AWS access key at the time of Edge-LB installation should have the following AWS API permissions in order to be able to manage Network Load Balancers:
+
+elasticloadbalancing:DescribeLoadBalancers
+elasticloadbalancing:CreateLoadBalancer
+elasticloadbalancing:DeleteLoadBalancer
+elasticloadbalancing:DescribeListeners
+elasticloadbalancing:CreateListener
+elasticloadbalancing:DeleteListener
+elasticloadbalancing:ModifyListener
+elasticloadbalancing:CreateTargetGroup
+elasticloadbalancing:DeleteTargetGroup
+elasticloadbalancing:DescribeTargetGroups
+elasticloadbalancing:ModifyTargetGroup
+elasticloadbalancing:RegisterTargets
+elasticloadbalancing:DeregisterTargets
+elasticloadbalancing:DescribeTargetHealth
+elasticloadbalancing:DescribeLoadBalancerAttributes
+elasticloadbalancing:ModifyLoadBalancerAttributes
+elasticloadbalancing:DescribeTags
+elasticloadbalancing:AddTags
+elasticloadbalancing:RemoveTags
+ec2:DescribeAddresses
+
+For more information about required permissions, see [Elastic Load Balancing API Permissions](https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/elb-api-permissions.html) in the AWS documentation. For more information about using Edge-LB configuration properties to provide AWS credentials, see the [Edge-LB configuration reference](https://github.com/mesosphere/dcos-edge-lb/blob/master/framework/edgelb/universe/config.json).
+
+# Deploying with the cloud provider load balancer
+To illustrate how you can deploy an Edge-LB pool that uses an Amazon Network Load Balancer, you can deploy a sample application that uses an Edge-LG pool to route incoming requests to the app.
+
+1. Open a text editor, then copy and paste the following sample app definition to create the `host-echo.json` file:
+
+  ```json
+  {
+      "id": "/host-echo",
+      "cmd": "/start $PORT0",
+      "instances": 1,
+      "cpus": 0.1,
+      "mem": 32,
+      "constraints": [["public_ip", "UNLIKE", "true"]],
+      "container": {
+          "type": "DOCKER",
+          "docker": {
+              "image": "mesosphere/echo-http"
+          }
+      },
+      "portDefinitions": [
+          {
+              "name": "web",
+              "protocol": "tcp",
+              "port": 0
+          }
+      ],
+      "healthChecks": [
+          {
+              "portIndex": 0,
+              "path": "/",
+              "protocol": "HTTP"
+          }
+      ]
+  }
+  ```
+
+1. Deploy the `host-echo` service using the `host-echo.json` app definition by running the following command:
```

Comment:
`1.` -> `2.`

Reply:
Needed to indent my code blocks. Fixed.

Reply:
I believe the auto-numbering feature of Markdown is used here.